src/Engine/GUI/Workspaces/OrbitalWorkspace.cpp

```
@@ -185,12 +200,53 @@ void OrbitalWorkspace::loadWorkspaceConfig(const std::string &configPath) {
+		}
+		
+		FilePathUtils::CreateFile(selectedFilePath);
+
+		FilePathUtils::WriteToFile(selectedFilePath, data);
+		m_simulationConfigPath = selectedFilePath;
+	}
+
+	m_simulationConfigSaved = true;
+
+	updateCodeEditorTitles();
 }
```

Cppcheck (2.20.0)

[style] The function 'severity' is never used.

(unusedFunction)